cardano-node/src/Cardano/Tracing/Config.hs

```diff
@@ -21,78 +28,185 @@ data TraceOptions
   | TracingOn TraceSelection
   deriving (Eq, Show)
 
+type TraceAcceptPolicy = ("TraceAcceptPolicy" :: Symbol)
```

Comment:
We might not need brackets here.

A vim macro to get rid of it :D
```
qqdi(va("-pq
```

Reply:
We don't need them for 8.10.2. I just added them so that it compiles with 8.6.3.

Reply:
Hmm, I didn't noticed that `GHC` parsers changed.